Specs/Core/GregorianDateSpec.js

```diff
@@ -0,0 +1,300 @@
+import { GregorianDate } from "../../Source/Cesium.js";
+
+describe("Core/GregorianDate", function () {
+  describe("Positive scenarios", function () {
+    it("Constructs any valid date", function () {
+      const validDate = new GregorianDate(2022, 2, 4, 23, 54, 0, 999.9, false);
+      expect(validDate.year).toEqual(2022);
+      expect(validDate.month).toEqual(2);
+      expect(validDate.day).toEqual(4);
+      expect(validDate.hour).toEqual(23);
+      expect(validDate.minute).toEqual(54);
+      expect(validDate.second).toEqual(0);
+      expect(validDate.millisecond).toEqual(999.9);
+      expect(validDate.isLeapSecond).toBeFalsy();
+    });
+
+    it("Constructs valid leap year date", function () {
+      const validDate = new GregorianDate(2024, 2, 29, 23, 54, 0, 999.9, false);
+      expect(validDate.year).toEqual(2024);
+      expect(validDate.month).toEqual(2);
+      expect(validDate.day).toEqual(29);
+      expect(validDate.hour).toEqual(23);
+      expect(validDate.minute).toEqual(54);
+      expect(validDate.second).toEqual(0);
+      expect(validDate.millisecond).toEqual(999.9);
+      expect(validDate.isLeapSecond).toBeFalsy();
+    });
+
+    it("Constructs the minimum date when no parameters are passed", function () {
+      const minimumDate = new GregorianDate();
+      expect(minimumDate.year).toEqual(1);
+      expect(minimumDate.month).toEqual(1);
+      expect(minimumDate.day).toEqual(1);
+      expect(minimumDate.hour).toEqual(0);
+      expect(minimumDate.minute).toEqual(0);
+      expect(minimumDate.second).toEqual(0);
+      expect(minimumDate.millisecond).toEqual(0);
+      expect(minimumDate.isLeapSecond).toBeFalsy();
+    });
+
+    it("Constructs valid dates for edge cases of days", function () {
+      expect(function () {
+        return new GregorianDate(2022, 1, 31);
+      }).not.toThrowDeveloperError();
+      expect(function () {
```

Comment:
Even though you test 2/28/2022 in the other tests, I think you should include it here for consistency.